programmers/큐.js

```diff
@@ -0,0 +1,101 @@
+class Queue {
+  constructor() {
+    this.data = [];
+    this.rear = 0;
+    this.size = 10000;
+  }
+
+  enqueue(element) {
+    if (this.rear < this.size) {
+      this.data[this.rear] = element;
+      this.rear = this.rear + 1;
+      return true;
+    }
+    // if there is no place to insert data
+    return false;
+  }
+
+  dequeue() {
+    if (this.isEmpty() === false) {
+      this.rear = this.rear - 1;
+      return this.data.shift();
+    }
+
+    return -1;
+  }
+
+  length() {
+    return this.rear;
+  }
+
+  isEmpty() {
+    return this.rear === 0;
+  }
+
+  getFront() {
+    if (this.isEmpty() === false) {
+      return this.data[0];
+    }
+
+    return -1;
+  }
+
+  getLast() {
+    if (this.isEmpty() === false) {
+      return this.data[this.rear - 1];
+    }
+
+    return -1;
+  }
+
+  print() {
+    for (let i = 0; i < this.rear; i++) {
+      console.log(this.data[i]);
+    }
+  }
+}
+
+let queue = new Queue();
+
+let solution = (inputLines) => {
```

Comment:
메인으로 들어가는 solution 함수는 이렇게 const로 선언해주세요!
```suggestion
const solution = (inputLines) => {
```